src/services/filterManager.ts

```diff
@@ -302,11 +302,11 @@ export const formatFromBasicSearch = (groups = [[]]) => {
 
       if (filter.operator === 'contains') {
         formattedFilter.bool.must.push({
-          regexp: { [filter.attribute]: '.*' + filter.value + '.*' }
+          regexp: { [filter.attribute]: '.*' + filter.value.toLowerCase() + '.*' }
```

Comment:
Interesting. I kinda remember we had the issue of case-sensitive Regex in Elasticsearch. Is it still the case?